src/utils/helperFunctions.ts

```diff
@@ -134,6 +134,58 @@ export const getChartDataFromTimeseriesData = (
     return chartData;
 };
 
+const getNDaysAverage = (
+    data: TimeseriesCountryDataRow[] | TimeseriesCaseCountsDataRow[],
+    idx: number,
+    nDays: number,
+    timeseriesCountData:
+        | TimeseriesCaseCountsDataRow[]
+        | TimeseriesCountryDataRow[],
+    cumulative = false,
+): number | null => {
+    //if first 7 days return null
+
+    if (
+        nDays > idx + 1 &&
+        compareAsc(data[0].date, timeseriesCountData[7].date) === -1
+    )
+        return null;
```

Comment:
You could return `undefined` here and this would mean that you don't have to declare types as `caseMovingNDaysCount?: number | null`. But that's just a suggestion